cnd/src/http_api/routes/rfc003/handlers/post_swap.rs

```diff
@@ -34,10 +34,10 @@ where
         Save<Request<AL, BL, AA, BA, AI, BI>> + Save<Accept<AI, BI>> + Save<Swap> + Save<Decline>,
     AL: Ledger,
     BL: Ledger,
-    AA: Ord + Send + Sync + 'static,
-    BA: Ord + Send + Sync + 'static,
-    AI: Send + Sync + 'static,
-    BI: Send + Sync + 'static,
+    AA: Clone + Ord + Send + Sync + 'static,
```

Comment:
As a note for reviewers: This clone appears here now because I had to move from references inside `HtlcParams` to owned values.
There would be a way of keeping references, so we don't need to clone stuff but it would spread lifetimes everywhere. I opt that having Clone here is the simpler approach.